SpatialOS/scripts/build-project.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+cd "$(dirname "$0")"
+
+echo $(pwd)
+
+WORKER_DIRS=(HelloWorker DiceWorker client)
+SDK_VERSION="13.5.1"
+BUILD_DIR="$(pwd)/../.."
+TOOLS_DIR="${BUILD_DIR}/SpatialOS/tools/${SDK_VERSION}"
+
+./download_dependencies.sh "${SDK_VERSION}"
```

Comment:
We now have the problem here that `download_dependencies.sh` cannot be run standalone without passing in the sdk version as an argument. How about creating a separate third file that defines the sdk version which is the imported by both this script and download_dependencies.sh?

Reply:
thanks for the suggestion! changing...